### PR TITLE
Task-40396 : Allow target attribute in HTML links

### PR DESCRIPTION
**Pull request by nidhalnidhal**
Before this fix , internal and external links in chat are opened in the same tab , so i refactored the behavior of v-sanitized-html directive in order to keep the target attribute on the external links so that they can be opened in a new tab.

### CODE REVIEW

**Comment by nidhalnidhal** [comment]
for security purposes , as opening a new tab may expose the platform to XSS attacks i added a "rel=noopener" tag to external links so that they wouldn't have a reference of the chat drawer window.

**Review by hbenali — COMMENTED** [review]
The chat module does not belong to the Meeds project. Please update the PR title and description (Remove all associated chat descriptions). We could use secure external link management as an example. cc @ahamdi  Thank you

**Comment by nidhalnidhal** [comment]
I already know that chat does not belong to the meeds project , and this PR is fixing a problem in the chat module , so i don't think it's causing any harm to mention it so that others could understand the reasons behind this change .

File: commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -518,7 +518,14 @@
...
           <include>/javascript/purify.min.js</include>
-          Vue.directive('sanitized-html',(el, binding) => el.innerHTML = DOMPurify.sanitize(Autolinker.link(binding.value, {newWindow: true}), {USE_PROFILES: {html: true}}));
+          Vue.directive('sanitized-html',(el, binding) =>
+          DOMPurify.addHook('afterSanitizeAttributes', function (node) {
+          // add noopener attribute to external links to eliminate vulnerabilities
+          if ('target' in node) {
+          node.setAttribute('rel', 'noopener');
+          }
+          });
+          el.innerHTML = DOMPurify.sanitize(Autolinker.link(binding.value, {newWindow: true}), {USE_PROFILES: {html: true},ADD_ATTR: ['target']}));
```
Comment (ahamdi) [inline commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml:528]:
```suggestion
          Vue.directive('sanitized-html',(el, binding) =>
            DOMPurify.addHook('afterSanitizeAttributes', function (node) {
              // add noopener attribute to external links to eliminate vulnerabilities
              if ('target' in node) {
                node.setAttribute('rel', 'noopener');
              }
          });
          el.innerHTML = DOMPurify.sanitize(Autolinker.link(binding.value, {newWindow: true}), {USE_PROFILES: {html: true},ADD_ATTR: ['target']}));
```

**Review by ahamdi — APPROVED** [review]

**Pull request merged**